release-notes/opensearch.release-notes-1.3.17.md

```diff
@@ -0,0 +1,6 @@
+## 2024-05-01 Version 1.3.17 Release Notes
```

Comment:
It should go to CHANGELOG.md , `[Unreleased 1.3.x]` section please (it will be moved to release notes upon release)

Reply:
Ah, my bad. Fixed.